app/Http/Controllers/Admin/CategoryController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Category;
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use \Validator;
+
+
+
+class CategoryController extends Controller
+{
+
+    /**
+     * Returns all categories
+     *
+     * @return \Illuminate\View\View
+     */
+    public function index()
```

Comment:
Why are categories here? We're doing auth. Also, the master doc clearly says that we're not doing categories